src/CAIP294.ts

```diff
@@ -120,6 +128,16 @@ function isValidAnnounceWalletEvent(
   );
 }
 
+/**
+ * Validates an {@link CAIP294Target} object.
```

Comment:
```suggestion
* Validates a {@link CAIP294Target} object.
```

nit ☝🏾 correct me if I'm wrong, but this seems like the proper way to "english" ?